libs/src/services/discoveryProvider/DiscoveryProvider.ts

```diff
@@ -928,6 +928,49 @@ export class DiscoveryProvider {
     return res.map((r) => ({ ...r, amount: parseInt(r.amount) }))
   }
 
+  /**
+   * Retrieves the user's replica se
```

Comment:
i think this should be left where it was under under internal functions and maybe just fix the "set" typo here too

Reply:
it seems like it's not an internal function so i think it makes more sense to leave above the internal functions?